pom.xml

```diff
@@ -46,23 +60,11 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <!-- TODO: set project ID -->
+          <!-- TODO: set project ID. -->
           <!-- <deploy.projectId>[PROJECT ID HERE]</deploy.projectId> -->
+          <deploy.projectId>step-2020-johndallard</deploy.projectId>
```

Comment:
Well... I also discussed a bit on this with Alvin but can we not set this for the open source version?

Reply:
Ack. Changed!